packages/suite/src/views/firmware/index.tsx

```diff
@@ -50,6 +51,8 @@ export const Firmware = ({ shouldSwitchFirmwareType }: FirmwareProps) => {
     const dispatch = useDispatch();
 
     const deviceModelInternal = device?.features?.internal_model;
+    const deviceWillBeWiped =
```

Comment:
nit: I guess it deserves a comment here - T2B1 get wiped switching between Universal and Bitcoin-only because different FW vendor headers as security feature

Reply:
e1268df4cf59d4a37b84b8a19e4f00d00d13254a